Cargo.toml

```diff
@@ -12,10 +12,13 @@ repository = "https://github.com/snapview/tungstenite-rs"
 version = "0.11.1"
 edition = "2018"
 
+[package.metadata.docs.rs]
+features = ["native-tls"]
```

Comment:
I'm thinking what `features` do we want to have here in the docs. Perhaps it would make sense to have `rustls-tls` instead? But then maybe it would make more sense to use `rustls-tls` as a default feature? @agalakhov, @strohel what do you think?

Reply:
@dnaka91 , was it meant to be `use-native-tls` instead of `native-tls`? :)

Reply:
> But then maybe it would make more sense to use rustls-tls as a default feature?

I think that would be a good future goal, but @dnaka91 mentioned
> the fact that webpki doesn't support IP SNIs yet makes it more likely to break clients if it would switch over.

So I would say not yet.

Reply:
Oh thanks, I forgot to update this. It was initially set like this because in the first version there could only be one of the two feature flags enabled at the same times.

Let's switch to enable all features, now that they can co-exist.